### PR TITLE
Adfs Module: Improve Inputs and Outputs for the AdfsClaimsProviderTrust Cmdlets

### PR DESCRIPTION
**Pull request by X-Guardian**
This PR improves the descriptions for the Inputs and Outputs section of the following `AdfsClaimsProviderTrust` cmdlets:

- Add-AdfsClaimsProviderTrust
- Disable-AdfsClaimsProviderTrust
- Enable--AdfsClaimsProviderTrust
- Get-AdfsClaimsProviderTrust
- Remove-AdfsClaimsProviderTrust
- Set-AdfsClaimsProviderTrust
- Update-AdfsClaimsProviderTrust

It also removes several explicit references to AD FS 2.0 and adds some missing related links.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/adfs/Add-AdfsClaimsProviderTrust.md [hunk 1]
```diff
@@ -682,14 +682,26 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+* The claims provider is responsible for collecting and authenticating a user's credentials, building up claims for that user, and packaging the claims into security tokens or Information Cards. In other words, a claims provider represents the organization for whose users the claims provider issues security tokens or Information Cards on their behalf. When you configure Active Directory Federation Services (AD FS) to use federation services, the role of the claims provider is to enable its users to access resources that a relying party organization hosts by establishing one side of a federation trust relationship. After the trust is established, tokens and Information Cards can be presented to the relying party across the federation trust.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Add-AdfsClaimsProviderTrust.md]:
```suggestion
* The claims provider is responsible for collecting and authenticating a user's credentials, building up claims for that user, and packaging the claims into security tokens or information cards. In other words, a claims provider represents the organization for whose users the claims provider issues security tokens or information cards on their behalf. When you configure Active Directory Federation Services (AD FS) to use federation services, the role of the claims provider is to enable its users to access resources that a relying party organization hosts by establishing one side of a federation trust relationship. After the trust is established, tokens and Information Cards can be presented to the relying party across the federation trust.
```

File: docset/windows/adfs/Enable-AdfsClaimsProviderTrust.md [hunk 2]
```diff
@@ -176,18 +176,31 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+* A relying party in Active Directory Federation Services (AD FS) is an organization in which Web servers that host one or more Web-based applications reside. Tokens and Information Cards that originate from a claims provider can then be presented and ultimately accessed by the Web-based resources that are located in the relying party organization. When AD FS 2.0 is configured in the role of the relying party, it acts as a partner that trusts a claims provider to authenticate users. Therefore, the relying party accesses the claims that are packaged in security tokens that come from users in the claims provider. In other words, a relying party is the organization whose Web servers are protected by the resource-side federation server. The federation server in the relying party uses the security tokens that the claims provider produces to issue tokens to the Web servers that are located in the relying party.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Enable-AdfsClaimsProviderTrust.md]:
```suggestion
* A relying party in Active Directory Federation Services (AD FS) is an organization in which web servers that host one or more web-based applications reside. Tokens and information cards that originate from a claims provider can then be presented and ultimately accessed by the web-based resources that are located in the relying party organization. When AD FS 2.0 is configured in the role of the relying party, it acts as a partner that trusts a claims provider to authenticate users. Therefore, the relying party accesses the claims that are packaged in security tokens that come from users in the claims provider. In other words, a relying party is the organization whose web servers are protected by the resource-side federation server. The federation server in the relying party uses the security tokens that the claims provider produces to issue tokens to the web servers that are located in the relying party.
```

File: docset/windows/adfs/Get-AdfsClaimsProviderTrust.md [hunk 3]
```diff
@@ -106,18 +106,27 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+* If you do not specify the *Name* parameter, the cmdlet lists all claims providers. The claims provider collects and authenticates a user's credentials, builds up claims for that user, and packages the claims into security tokens or Information Cards. In other words, a claims provider represents the organization for whose users the claims provider issues security tokens or Information Cards on their behalf. When you configure Active Directory Federation Services (AD FS), the role of the claims provider is to enable its users to access resources that are hosted in a relying party organization by establishing one side of the federation trust relationship. After the federation trust is established, tokens and Information Cards can be presented to the relying party across the trust.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Get-AdfsClaimsProviderTrust.md]:
```suggestion
* If you do not specify the *Name* parameter, the cmdlet lists all claims providers. The claims provider collects and authenticates a user's credentials, builds up claims for that user, and packages the claims into security tokens or information cards. In other words, a claims provider represents the organization for whose users the claims provider issues security tokens or information cards on their behalf. When you configure Active Directory Federation Services (AD FS), the role of the claims provider is to enable its users to access resources that are hosted in a relying party organization by establishing one side of the federation trust relationship. After the federation trust is established, tokens and information cards can be presented to the relying party across the trust.
```

File: docset/windows/adfs/Set-AdfsClaimsProviderTrust.md [hunk 4]
```diff
@@ -779,18 +779,35 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
...
+X509Certificate2 objects are received by the *TargetCertificate* parameter.
+
 ### Microsoft.IdentityServer.PowerShell.Resources.ClaimsProviderTrust
-A class structure that represents a claims provider trust.
+
+ClaimsProviderTrust objects are received by the *TargetClaimsProviderTrust* parameter.
+
+### System.String
+
+String objects are received by the *AcceptanceTransformRules*, *SamlEndpoint*, *TargetIdentifier* and *TargetName* parameters.
```
Comment (JohanFreelancer9) [inline docset/windows/adfs/Set-AdfsClaimsProviderTrust.md]:
```suggestion
String objects are received by the *AcceptanceTransformRules*, *SamlEndpoint*, *TargetIdentifier*, and *TargetName* parameters.
```

**Comment by e0i** [comment]
@get-itips Is there any additional comments for this PR? Thanks.

**Pull request merged**